components/dialogs/AuthDialog.vue

```diff
@@ -32,7 +32,16 @@
       slot="header-center"
       class="auth-dialog__header-center"
     >
-      <img :src="LikeCoinLogo">
+      <img
+        v-if="avatar"
+        :src="avatar"
+        class="auth-dialog__avatar auth-dialog__avatar--user"
+      >
+      <img
+        v-else
+        :src="LikeCoinLogo"
+        class="auth-dialog__avatar auth-dialog__avatar--likecoin"
```

Comment:
I prefer to name this class as `auth-dialog__logo`, with modifier `--avatar` and `--likecoin`.

I suggest to add a computed property i.e. `logoProps` which return
```javascript
{
  src: this.shouldShowFromUserIcon ? this.avatar : LikeCoinLogo,
  class: `auth-dialog__logo auth-dialog__logo--${this.shouldShowFromUserIcon ? 'avatar' : 'likecoin'}`,
}
```
so we can use `v-bind="logoProps"` with one `<img/>`